frontend/ui/elements/common_settings_menu_table.lua

```diff
@@ -40,6 +41,10 @@ common_settings.screen = {
         require("ui/elements/refresh_menu_table"),
     },
 }
+if Device:isAndroid() then
+    table.insert(common_settings.screen.sub_item_table, require("ui/elements/screen_fullscreen_menu_table"))
```

Comment:
@Frenzie Just to be sure if it's failing or not I changed the line 44 to _if not Device:isAndroid()_ (to test on PC).
And it fails:

> ./luajit: frontend/device/android/device.lua:16: missing declaration for symbol 'AConfiguration_getDensity'
> stack traceback:
>         [C]: in function '__index'
>         frontend/device/android/device.lua:16: in main chunk
>         [C]: in function 'require'
>         frontend/ui/elements/screen_fullscreen_menu_table.lua:4: in main chunk
>         [C]: in function 'require'
>         frontend/ui/elements/common_settings_menu_table.lua:45: in main chunk
>         [C]: in function 'require'
>         frontend/apps/filemanager/filemanagermenu.lua:161: in function 'old_method'
>         frontend/dbg.lua:43: in function 'setUpdateItemTable'
>         frontend/apps/filemanager/filemanagermenu.lua:327: in function 'onShowMenu'
>         frontend/apps/filemanager/filemanagermenu.lua:375: in function 'handler'
>         frontend/ui/widget/container/inputcontainer.lua:197: in function 'handleEvent'
>         frontend/ui/widget/container/widgetcontainer.lua:87: in function 'propagateEvent'
>         frontend/ui/widget/container/widgetcontainer.lua:105: in function 'handleEvent'
>         frontend/ui/uimanager.lua:446: in function 'sendEvent'
>         frontend/ui/uimanager.lua:42: in function '__default__'
>         frontend/ui/uimanager.lua:722: in function 'handleInput'
>         frontend/ui/uimanager.lua:767: in function 'run'
>         ./reader.lua:193: in main chunk
>         [C]: at 0x00404750
> 

not sure what's wrong there.